src/services/get-todo-list-service.ts

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+export const getToDoList = async () => {
+  const response = await axios.get(`${process.env.REACT_APP_BASE_URL}/todos`);
+
+  return response.data;
```

Comment:
poderia usar desestruturação assim pra ficar mais clean:
```suggestion
  const { data } = await axios.get(`${process.env.REACT_APP_BASE_URL}/todos`);

  return data;
```